ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -6,8 +6,8 @@ Resource       ../ODHDataSciencePipelines.resource
 
 *** Variables ***
 ${PIPELINES_SECTION_XP}=    xpath://div[@id="pipelines-projects"]
-${PIPELINES_SERVER_BTN_XP}=    xpath://button[text()="Create a pipeline server"]
-${PIPELINES_SERVER_CONFIG_BTN_XP}=    xpath://button[text()="Configure"]
+${PIPELINES_SERVER_BTN_XP}=    xpath://button[text()="Configure pipeline server"]
+${PIPELINES_SERVER_CONFIG_BTN_XP}=    xpath://footer/button[text()="Configure pipeline server"]
```

Comment:
Is the idea of having /footer to use the other button that contains the same text?

Reply:
@diegolovison Yes, I need to distinguish between the two, because when I am clicking the `footer/` one, the original one is still around and I must not click that.